### PR TITLE
feat(jinja2): add jinja2

### PR DESCRIPTION
**Pull request by cathaysia**
add https://github.com/cathaysia/tree-sitter-jinja2

### CODE REVIEW

File: queries/queries/jinja2 [hunk 1]
```diff
@@ -0,0 +1 @@
+../tree-sitter-jinja2/queries
```
Comment (lucario387) [inline queries/queries/jinja2:1]: uhhh ...

File: lua/nvim-treesitter/parsers.lua [hunk 2]
```diff
@@ -1021,6 +1021,24 @@ list.javascript = {
   maintainers = { "@steelsojka" },
 }
 
+list.jinja2 = {
+  install_info = {
+    url = "https://github.com/cathaysia/tree-sitter-jinja2",
+    location = "tree-sitter-jinja2",
+    files = { "src/parser.c", "src/scanner.c" },
+  },
+  maintainers = { "@steelsojka" },
```
Comment (clason) [inline lua/nvim-treesitter/parsers.lua:1030]: No.

**Comment by clason** [comment]
Make sure your repo does not contain any symlinks; these don't work on Windows so we can't include the parser.

**Review by clason — CHANGES_REQUESTED** [review]
You need to use only the captures listed in CONTRIBUTING.MD.

**Pull request closed without merging**